packages/frontend/src/utils/getSearch.ts

```diff
@@ -0,0 +1,13 @@
+import { IUser } from 'src/types';
+
+export const getSearch = (users: IUser[], value?: string) => {
+  if (users.length === 0 || !value) return users;
+
+  return users.filter(
```

Comment:
Let's cast value to lowerCase too to make search case-insensitive?

Reply:
Done